README.md

```diff
@@ -1 +1,3 @@
-# MarkdownTest
\ No newline at end of file
+# MarkdownTest
+
+##Test 2
```

Comment:
I'm not sure this is a good idea.
![image](https://cloud.githubusercontent.com/assets/6132506/18689156/c5b8a272-7f55-11e6-83b3-ffdf532310de.png)